chainservice/chainservice.go

```diff
@@ -83,6 +108,16 @@ func (cs *ChainService) HandleAction(ctx context.Context, actPb *iotextypes.Acti
 	return err
 }
 
+func chainIDmetrics(act action.SealedEnvelope) {
+	chainID := strconv.FormatUint(uint64(act.ChainID()), 10)
+	if act.ChainID() > 0 {
+		_apiCallWithChainIDMtc.WithLabelValues(chainID).Inc()
+	} else {
+		recipient, _ := act.Destination()
```

Comment:
it will be empty for staking action, change string to `staking` in such case

Reply:
done